drivers/windows/thread_windows.cpp

```diff
@@ -78,11 +78,29 @@ void ThreadWindows::wait_to_finish_func_windows(Thread *p_thread) {
 	CloseHandle(tp->handle);
 	//`memdelete(tp);
 }
+typedef HRESULT(WINAPI *PSETTHREADDESCRIPTION)(HANDLE hThread, PCWSTR lpThreadDescription);
+static PSETTHREADDESCRIPTION lpSetThreadDescription;
+
+Error ThreadWindows::set_name_func_windows(const String &p_name) {
+	ERR_FAIL_NULL_V(lpSetThreadDescription, ERR_UNAVAILABLE);
+
+	HRESULT hr = (*lpSetThreadDescription)(GetCurrentThread(), p_name.c_str());
+	ERR_FAIL_COND_V(FAILED(hr), ERR_INVALID_PARAMETER);
+	return OK;
+}
 
 void ThreadWindows::make_default() {
 	create_func = create_func_windows;
 	get_thread_id_func = get_thread_id_func_windows;
 	wait_to_finish_func = wait_to_finish_func_windows;
+
+	// SetThreadDescription is implemented in Windows 10, version 1607.
+	lpSetThreadDescription = (PSETTHREADDESCRIPTION)GetProcAddress(
+					GetModuleHandle(TEXT("kernel32.dll")),
```

Comment:
You need to use only 2 tabs on the extra lines in the function call to fix clang format errors.